terraform/mgmtcluster.tf

```diff
@@ -111,10 +152,16 @@ runcmd:
   - sysctl -w -p /etc/sysctl.d/90-conntrack_max.conf
   - mkdir /etc/docker
   - HOME=/home/${var.ssh_username} /tmp/get_mtu.sh
-  - mv /tmp/daemon.json /etc/docker/daemon.json
+  - cp /tmp/daemon.json /etc/docker/daemon.json
```

Comment:
Could you please describe why the move command needed to be changed here?

Reply:
i think that was for debugging purposes only, changed it back